front-end/src/UI/Templates/InviteGame/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import {
+  Grid,
+  GridItem,
+  VStack,
+  Text,
+  Button,
+  Input,
+  Menu,
+  MenuButton,
+  MenuList,
+  MenuItem,
+} from '@chakra-ui/react';
+import { Icon, ChevronDownIcon } from '@chakra-ui/icons';
+import { GrGamepad } from 'react-icons/gr';
+
+function Invitegame() {
```

Comment:
디렉토리명과 컴포넌트 이름이 달라유.

```suggestion
function InviteGame() {
```

해주세요.